pkgs/by-name/lc/lcevcdec/package.nix

```diff
@@ -42,16 +42,26 @@ stdenv.mkDerivation (finalAttrs: {
     })
   ];
 
-  postPatch = ''
-    substituteInPlace cmake/tools/version_files.py \
-      --replace-fail "args.git_version" '"${finalAttrs.version}"' \
-      --replace-fail "args.git_hash" '"${finalAttrs.src.rev}"' \
-      --replace-fail "args.git_date" '"1970-01-01"'
-  '';
+  postPatch =
+    ''
+      substituteInPlace cmake/tools/version_files.py \
+        --replace-fail "args.git_version" '"${finalAttrs.version}"' \
+        --replace-fail "args.git_hash" '"${finalAttrs.src.rev}"' \
+        --replace-fail "args.git_date" '"1970-01-01"'
+
+    ''
+    + lib.optionalString (!stdenv.hostPlatform.avx2Support) ''
+      substituteInPlace cmake/modules/Compiler/GNU.cmake \
+        --replace-fail "-mavx" ""
```

Comment:
This smells like it'd disable *any* AVX support, not just AVX2 which mismatches with the guard.

Reply:
Yes indeed, and that should be avxSupport and not avx2Support